cmake/SetupSpheral.cmake

```diff
@@ -85,7 +90,10 @@ if(ENABLE_CUDA)
   #set(CMAKE_CUDA_FLAGS  "${CMAKE_CUDA_FLAGS} -arch=${CUDA_ARCH} --expt-relaxed-constexpr --extended-lambda -Xcudafe --display_error_number")
   set(CMAKE_CUDA_STANDARD 17)
   list(APPEND SPHERAL_CXX_DEPENDS cuda)
+  set(SPHERAL_ENABLE_CUDA On)
+  set(SPHERAL_ENABLE_VVI On)
```

Comment:
Should the Value-View interface be enabled for CPU builds as well?

Reply:
It *can* be used for CPU builds but it is *required* for any GPU builds where we will be executing on the device. This just ensures it's enabled if you are building spheral w/ cuda.